toolchain/driver/build_runtimes_subcommand.cpp

```diff
@@ -0,0 +1,86 @@
+// Part of the Carbon Language project, under the Apache License v2.0 with LLVM
+// Exceptions. See /LICENSE for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#include "toolchain/driver/build_runtimes_subcommand.h"
+
+#include "llvm/TargetParser/Triple.h"
+#include "toolchain/driver/clang_runner.h"
+
+namespace Carbon {
+
+auto BuildRuntimesOptions::Build(CommandLine::CommandBuilder& b) -> void {
+  b.AddStringOption(
+      {
+          .name = "output-directory",
+          .value_name = "DIR",
+          .help = R"""(
+The directory to populate with runtime libraries suitable for the selected code
+generation options.
+)""",
+      },
+      [&](auto& arg_b) { arg_b.Set(&directory); });
+
+  codegen_options.Build(b);
+}
+
+static constexpr CommandLine::CommandInfo SubcommandInfo = {
+    .name = "build-runtimes",
+    .help = R"""(
+Build Carbon's runtime libraries.
+
+This subcommand builds Carbon's runtime libraries for a particular code
+generation target. These can be built into their default location or into a
+specified location.
+
+Directly running this command is not necessary as Carbon will work to build the
+runtimes needed on-demand when linking, but directly building them can aid in
+debugging issues or allow building them with a specific set of code generation
+flags and then reusing them in links explicitly.
```

Comment:
Some minor copy-editing:
```suggestion
This subcommand builds Carbon's runtime libraries for a particular code
generation target, either in their default location or a specified one.

Running this command directly is not necessary as Carbon will build and cache
runtimes as needed when linking, but building them directly can aid in
debugging issues or allow them to be prebuilt, possibly with customized code
generation flags, and used explicitly when linking.
```